lib/Twig/Cache/Null.php

```diff
@@ -16,12 +16,14 @@
  */
 class Twig_Cache_Null implements Twig_CacheInterface
 {
+    private $buffer = array();
```

Comment:
Nope, this is 1.x, still supporting older versions of PHP. But anyway, I don't plan to migrate to the new short array syntax, mainly because I'm going to maintain both 1.x and 2.x in parallel, so ease of merge is a must.